src/main/resources/headers/cudnn_header.h

```diff
@@ -0,0 +1,10 @@
+#include <cudnn.h>
+
+#define CUDNNCHECK(cmd) do {                         \
+  cudnnStatus_t status = cmd;                       \
+  if( status != CUDNN_STATUS_SUCCESS ) {            \
+    printf("Cudnn failure %s:%d '%s'\n",            \
+        __FILE__,__LINE__,cudnnGetErrorString(cmd));   \
+    exit(EXIT_FAILURE);                             \
+  }                                                 \
+} while(0)
```

Comment:
GitHub wants a new line by the end of the file

Reply:
fixed in 7be0fe